code/datums/emotes.dm

```diff
@@ -201,3 +203,16 @@
 				ghost.show_message("[FOLLOW_LINK(ghost, src)] [ghost_text]")
 
 	visible_message(text, visible_message_flags = list(CHATMESSAGE_EMOTE = TRUE))
+
+/**
+ * Returns a boolean based on whether or not the string contains a comma or an apostrophe,
+ * to be used for emotes to decide whether or not to have a space between the name of the user
```

Comment:
I'm really confused in what case you would NOT want a space

Reply:
Removed the Proc and made it the way it was previously

Reply:
*me's eyeballs FUCKING EXPLODE gets spit out as `Kapu1178 's eyeballs FUCKING EXPLODE` without this.

Reply:
Ooookay time to reinstall it then!

Reply:
I just want someone to explain

Reply:
Oh kapu explained. i thought it was a meme